### PR TITLE
Send update_type to put_content

### PR DESCRIPTION
**Pull request by emmabeynon**
We are deprecating sending the `update_type` in publish requests made to the Publishing API, and now expect clients to send `update_type` in PUT requests instead. We also upgrade gds-api-adapters to bring this app up to date with the new expected behaviour.

https://trello.com/c/LnJlkb9e/987-5-deprecate-the-usage-of-updatetype-in-publish-for-publishing-api-and-update-usage-across-govuk

### CODE REVIEW

File: app/models/guide_republisher.rb [hunk 1]
```diff
@@ -12,7 +12,7 @@ def republish
 
       publishing_api.put_content(guide.content_id, guide_for_publication.content_payload)
```
Comment (36degrees) [inline app/models/guide_republisher.rb]: Should this have had 'minor' moved to here, or am I not understanding this change?

Comment (thomasleese) [inline app/models/guide_republisher.rb]: It's available in `guide_for_publication.content_payload`.

Comment (36degrees) [inline app/models/guide_republisher.rb]: But this script is meant to re-publish guides but always as a minor edition, as we use this script if e.g. we need to change the way a guide is presented, but we don't want to trigger email alerts or show a change in the history of the guide.

Comment (thomasleese) [inline app/models/guide_republisher.rb]: Ah sure, we'll revert this change and use the `republish` `update_type` here.

Comment (36degrees) [inline app/models/guide_republisher.rb]: Thanks, appreciate it 👍

Comment (emmabeynon) [inline app/models/guide_republisher.rb]: Thanks for explaining @36degrees - PR is open to fix this here https://github.com/alphagov/service-manual-publisher/pull/399
